flowzone.yml

```
@@ -499,6 +497,15 @@
       echo "enforce_admins__enabled=$(jq -cr '.enforce_admins.enabled // false' <<< "${result}")" >> $GITHUB_OUTPUT
       echo "block_creations__enabled=$(jq -cr '.block_creations.enabled // false' <<< "${result}")" >> $GITHUB_OUTPUT
 
+  - &sortNodeVersions
+    name: Sort node version
+    id: node_versions
+    env:
+      VERSIONS: ${{ needs.is_npm.outputs.node_versions }}
+    run: |
+      echo "min=$(echo "${VERSIONS}" | jq -r '.[]' | sort -V | head -n1)" >> $GITHUB_OUTPUT
+      echo "max=$(echo "${VERSIONS}" | jq -r '.[]' | sort -V -r | head -n1)" >> $GITHUB_OUTPUT
```

In general I prefer long flags for committed scripts so that it's easier for people who are later reading it and may not be as familiar with the short flags:
```suggestion
      echo "min=$(echo "${VERSIONS}" | jq -r '.[]' | sort --version-sort | head -n1)" >> $GITHUB_OUTPUT
      echo "max=$(echo "${VERSIONS}" | jq -r '.[]' | sort --version-sort --reverse | head -n1)" >> $GITHUB_OUTPUT
```